layouts/calendars/list.html

```diff
@@ -36,6 +36,46 @@ <h5 class="subtitle is-5 is-muted">{{ .Params.Subtitle }}</h5>
           {{ end }}
         </div>
 
+        <br>
+        <script src="https://cdn.jsdelivr.net/npm/vega@5"></script>
+        <link href='https://cdn.jsdelivr.net/npm/fullcalendar@5.11.0/main.min.css' rel='stylesheet' />
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/ical.js/1.4.0/ical.min.js"></script>
+        <script src='https://cdn.jsdelivr.net/npm/fullcalendar@5.11.0/main.min.js'></script>
+        <script src="https://cdn.jsdelivr.net/npm/@fullcalendar/icalendar@5.11.0/main.global.min.js"></script>
+
+        <script>
+          document.addEventListener('DOMContentLoaded', function() {
+            var cal_colors = vega.scheme('category10');
+            var community_calendar = document.getElementById('community_calendar');
+            var calendar = new FullCalendar.Calendar(community_calendar, {
+                timeZone: 'local',
+                initialView: 'dayGridMonth',
+                headerToolbar: {
+                  left: "prev,next today",
+                  center: "title",
+                  right: "dayGridMonth,timeGridWeek,timeGridDay",
+                },
+                eventSources: [
+                {{ range $key, $value := $.Site.Data.calendars }}
+                    {
+                        url: '{{ $key }}.ics',
+                        format: 'ics',
+                        backgroundColor: cal_colors.shift(),
+                    },
+                {{ end }}
```

Comment:
I am not sure about the syntax please double check 😅
```suggestion
                {{ range $key, $value := $.Site.Data.calendars }}
                    let cal_color = cal_colors.shift()
                    {
                        url: '{{ $key }}.ics',
                        format: 'ics',
                        backgroundColor: cal_color,
                        border-color: cal_color;
                    },
                {{ end }}
```